builder/ibmcloud/vpc/config.go

```diff
@@ -31,6 +31,7 @@ type Config struct {
 	ResourceGroupID           string `mapstructure:"resource_group_id"`
 	SecurityGroupID           string `mapstructure:"security_group_id"`
 	VSIBaseImageID            string `mapstructure:"vsi_base_image_id"`
+	VSIBootVolumeID           string `mapstructure:"vsi_boot_volume_id"`
```

Comment:
Suggestion to align it alphabetically.

Reply:
Moved VSIBootVolumeID below VSIBaseImageName